pkg/providers/k8s/secret.go

```diff
@@ -400,8 +400,8 @@ func (c *secretController) onUpdate(prev, curr interface{}) {
 		return
 	}
 	log.Debugw("secret update event arrived",
-		zap.Any("new object", curr),
-		zap.Any("old object", prev),
+		zap.Any("new object", curr.(*corev1.Secret).ObjectMeta.Name),
+		zap.Any("old object", prev.(*corev1.Secret).ObjectMeta.Name),
```

Comment:
we don't need to log secret name twice since it is immutable

Reply:
removed